ambuda/views/proofing/project.py

```diff
@@ -104,6 +104,31 @@ def summary(slug):
     )
 
 
+@bp.route("/<slug>/activity")
+def activity(slug):
+    """Download the project in various output formats."""
+    project_ = q.project(slug)
+    if project_ is None:
+        abort(404)
+
+    session = q.get_session()
+    recent_revisions = (
+        session.query(db.Revision)
+        .filter_by(project_id=project_.id)
+        .order_by(db.Revision.created.desc())
+        .limit(100)
+        .all()
+    )
+    recent_activity = [("revision", r.created, r) for r in recent_revisions]
```

Comment:
Do we think we need all the information stored in a revision or project? Can we override their `__str__` methods to only provide some relevant info for users? 

Related to this, adding screenshots of the end result would be helpful :)

Reply:
Oh never mind, it looks like it's already doing this. I've added a screenshot to showcase: 
<img width="541" alt="Screen Shot 2022-08-31 at 11 58 23 PM" src="https://user-images.githubusercontent.com/10256087/187851626-fd44bc8c-80c3-428d-88cf-54dff47de4a2.png">

Reply:
No, we don't need all of it or even most of it, but I think right now that's a premature optimization. The only one that's clearly useless is `revision.content` which is also big, so I added `defer`s to all of the queries for that.